### PR TITLE
Feat/NFT detail page

### PR DESCRIPTION
**Pull request by abbasabid**
# Description of change

NFT detail page design with mock data

## Type of change

- Enhancement (a non-breaking change which adds functionality)

### CODE REVIEW

File: client/src/app/components/stardust/Activity.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { FunctionComponent } from 'react';
```
Comment (msarcev) [inline client/src/app/components/stardust/Activity.tsx:1]: Lets just use double-quotes `"` in imports everywhere.

File: client/src/app/components/stardust/Activity.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import "./Transaction.scss";
+import { ActivityProps } from "./ActivityProps";
+
+export const Activity: FunctionComponent<ActivityProps> = ({ hash, date, action, status, price, tableFormat  }) =>
```
Comment (msarcev) [inline client/src/app/components/stardust/Activity.tsx:5]: Let's use `React.FC<ActivityProps>` so we have less imports.

File: client/src/app/components/stardust/Asset.tsx [hunk 3]
```diff
@@ -1,57 +1,43 @@
-import React, { Component, ReactNode } from "react";
+import { FunctionComponent } from 'react';
```
Comment (msarcev) [inline client/src/app/components/stardust/Asset.tsx:1]: Lets just use double-quotes " in imports everywhere.

File: client/src/app/components/stardust/Asset.tsx [hunk 4]
```diff
@@ -1,57 +1,43 @@
...
-                        </div>
-                    </div>
-                </div >
-            )
-        );
-    }
-}
-
-export default Asset;
+export const Asset: FunctionComponent<AssetProps> = ({ asset, symbol, quantity, price, value, tableFormat  }) =>
```
Comment (msarcev) [inline client/src/app/components/stardust/Asset.tsx:5]: Let's use `React.FC<AssetProps>` so we have less imports.

File: client/src/app/components/stardust/NFT.tsx [hunk 5]
```diff
@@ -1,28 +1,23 @@
-import React, { Component, ReactNode } from "react";
+import { FunctionComponent } from 'react';
```
Comment (msarcev) [inline client/src/app/components/stardust/NFT.tsx:1]: Double quotes.

File: client/src/app/components/stardust/NFT.tsx [hunk 6]
```diff
@@ -1,28 +1,23 @@
...
-                />
-                <span className="nft-name">Token: {this.props.tokenName}</span>
-                <span className="nft-id">Token ID: {this.props.tokenID}</span>
-            </div>
-        );
-    }
-}
-
-export default NFT;
+export const NFT: FunctionComponent<NFTProps> = ({ image, tokenName, tokenID, network  }) =>
```
Comment (msarcev) [inline client/src/app/components/stardust/NFT.tsx:6]: React.FC

**Review by msarcev — CHANGES_REQUESTED** [review]
Other than this looks good. Also do `npm run lint && npm run sass-lint` check please.

File: client/src/app/components/stardust/Activity.tsx (hunk 1 above)
Comment (abbasabid) [reply client/src/app/components/stardust/Activity.tsx]: Done

File: client/src/app/components/stardust/Activity.tsx (hunk 2 above)
Comment (abbasabid) [reply client/src/app/components/stardust/Activity.tsx]: Done

File: client/src/app/components/stardust/Asset.tsx (hunk 3 above)
Comment (abbasabid) [reply client/src/app/components/stardust/Asset.tsx]: Done

File: client/src/app/components/stardust/Asset.tsx (hunk 4 above)
Comment (abbasabid) [reply client/src/app/components/stardust/Asset.tsx]: Done

File: client/src/app/components/stardust/NFT.tsx (hunk 5 above)
Comment (abbasabid) [reply client/src/app/components/stardust/NFT.tsx]: Done

File: client/src/app/components/stardust/NFT.tsx (hunk 6 above)
Comment (abbasabid) [reply client/src/app/components/stardust/NFT.tsx]: Done

**Review by msarcev — APPROVED** [review]

**Pull request merged**